sig-auth/annual-report-2022.md

```diff
@@ -62,19 +82,17 @@
 
 ## Membership
 
-- Primary slack channel member count:
-- Primary mailing list member count:
-- Primary meeting attendee count (estimated, if needed):
-- Primary meeting participant count (estimated, if needed):
+- Primary slack channel member count: 2847
+- Primary mailing list member count: 462
+- Primary meeting attendee count (estimated, if needed): 20 ~ 30
+- Primary meeting participant count (estimated, if needed): 5 ~ 10
 - Unique reviewers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
 - Unique approvers for SIG-owned packages: <!-- in future, this will be generated from OWNERS files referenced from subprojects, expanded with OWNERS_ALIASES files -->
 
 Include any other ways you measure group membership
 
 ## [Subprojects](https://git.k8s.io/community/sig-auth#subprojects)
 
-
-
 **New in 2022:**
 
   - sig-auth-tools
```

Comment:
should https://github.com/kubernetes-sigs/pspmigrator be added?

Reply:
this list was auto populated by the tool and I think `pspmigrator` wasn't added because it's not part of https://github.com/kubernetes/community/tree/master/sig-auth. I'll add it to the list here and we'll need a follow-up PR to add it to the sig-auth list.

Reply:
> Does pspmigrator fall under Policy Management as per https://github.com/kubernetes/community/tree/master/sig-auth#policy-management?

yes, it's already listed under that section.

Reply:
do we want to explicitly call out `pspmigrator` here or just leave it as-is?

Reply:
I _think_ the intention is to add it here because it's something new that got added in 2022.

Reply:
added!